db/seeds.rb

```diff
@@ -20,4 +20,77 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome"
+  )
+
+ParticleDevice.create!(
```

Comment:
Because Rhizome has `accepts_nested_attributes_for :particle_device` , you should be able to scoot all this ParticleDevice stuff up into the Rhizome #create (without the rhizome_id) and it'll create and associate the device. It is set up that way specifically because the Particle device (e.g. the Photon) is an integral component of the Rhizome and it would be a good reminder to set up the seed that way.